ang/crmUi.js

```diff
@@ -642,7 +642,7 @@
             $timeout(function () {
               // ex: msg_template_id adds new item then selects it; use $timeout to ensure that
               // new item is added before selection is made
-              element.select2('val', ngModel.$viewValue);
+              element.select2('val', ngModel.$modelValue);
```

Comment:
It looks like the only other usages are in the CiviMail UI - with the (1) "Message Template" dropdown, (2) "From/Reply-To" dropdown, and (3) the "Recipient" advanced options. We should be fine if those three work.